.gitignore

```diff
@@ -12,3 +12,4 @@
 /silverstripe-dynamodb/
 /crontask/
 /quickfeedback/
+raygun/
```

Comment:
should this not be `/raygun/` (with a leading `/`)?

Reply:
Yes, good catch. Hotfixed with 554070a